pkg/kubelet/kubelet.go

```diff
@@ -1607,7 +1607,7 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 	pullSecrets := kl.getPullSecretsForPod(pod)
 
 	// Call the container runtime's SyncPod callback
-	result := kl.containerRuntime.SyncPod(pod, apiPodStatus, podStatus, pullSecrets, kl.backOff)
+	result := kl.containerRuntime.SyncPod(pod, existingStatus, podStatus, pullSecrets, kl.backOff)
```

Comment:
See my comments here: https://github.com/kubernetes/kubernetes/pull/45682#issuecomment-301137582

The SyncPod function was ignoring the second argument, I'm not sure why apiPodStatus was being sent to begin with.  (apiPodStatus appears to be the status object that will be sent back to the API server to report state... existingStatus is the status that the kubelet knew about from the last time it checked.)

Since we need the previous status plumbed through, and SyncPod wasn't using the parameter before, I re-used that positional parameter rather than adding a new one.